packages/web/src/scss/components/Toast/README.md

```diff
@@ -178,33 +182,75 @@ An icon can be added to the ToastBar component:
 ```html
 <div class="ToastBar ToastBar--inverted">
   <div class="ToastBar__box">
-    <div class="ToastBar__content">
+    <div class="ToastBar__container">
       <svg width="20" height="20" aria-hidden="true">
         <use xlink:href="/icons/svg/sprite.svg#info" />
       </svg>
-      <div class="ToastBar__message">Message with icon</div>
+      <div class="ToastBar__content">
+        <div class="text-truncate-multiline" data-spirit-populate-field="message">Message with icon</div>
+      </div>
+    </div>
+  </div>
+</div>
+```
+
+### ToastBar Components
+
+The content of `ToastBar` can be assembled from the following components:
+
+#### ToastBarMessage
+
+`ToastBarMessage` is a component designates for main message in `ToastBar`.
+
+Usage example:
+
+```html
+<div class="ToastBar ToastBar--inverted">
+  <div class="ToastBar__box">
+    <div class="ToastBar__container">
+      <div class="ToastBar__content">
+        <div class="text-truncate-multiline" data-spirit-populate-field="message">Message with action</div>
+      </div>
     </div>
   </div>
 </div>
 ```
 
-### Action Link
+#### API
+
+| Name       | Type     | Default | Required | Description                    |
+| ---------- | -------- | ------- | -------- | ------------------------------ |
+| `children` | `string` | —       | ✓        | Content of the ToastBarMessage |
+
+#### ToastBarLink
 
-An action link can be added to the ToastBar component:
+`ToastBarLink` is a component designated to create an action link within `ToastBar`.
```

Comment:
```suggestion
`ToastBarLink` is a subcomponent designated to create an action link within `ToastBar`.
```

Reply:
I am not sure if we really do this everywhere. If not, please ignore my suggestion